src/main/app/src/app/taken/taak-view/taak-view.component.ts

```diff
@@ -145,19 +149,18 @@ export class TaakViewComponent extends AbstractView implements OnInit, AfterView
 
     editGroep(event: any): void {
         this.taak.groep = event.groep;
-
-        this.takenService.assignGroup(this.taak).subscribe(taak => {
-            this.utilService.openSnackbar('msg.taak.toegekend', {behandelaar: taak.groep.naam});
-            this.init(taak);
+        this.takenService.assignGroup(this.taak).subscribe(() => {
+            this.utilService.openSnackbar('msg.taak.toegekend', {behandelaar: this.taak.groep.naam});
+            this.init(this.taak);
```

Comment:
is dit wel handig? Wil je niet een geupdate taak van de server hebben?

Reply:
Voor een taak worden er veel uitstapjes gemaakt naar flowable; (procesinstance, zaaktypeuuid). Ook worden de zaakafhandelparamters opgehaaald(formulierdefinitie). maar er zijn geen wijzigingen op de taak die je terug krijgt, deze is exact hetzelfde als je heen stuurt, en eventuele  exceptions krijg je ook wel van een void.